app/serializers/search/listing_result_serializer.rb

```diff
@@ -0,0 +1,29 @@
+module Search
+  # TODO: [@atsmith813] Rename this to just ListingSerializer once Elasticsearch
+  # is fully removed
+  class ListingResultSerializer < ApplicationSerializer
+    attributes :id,
+               :body_markdown,
+               :bumped_at,
+               :category,
+               :contact_via_connect,
+               :expires_at,
+               :originally_published_at,
+               :location,
+               :processed_html,
+               :published,
+               :slug,
+               :title,
+               :user_id
+
+    attribute :tags do |cl|
```

Comment:
Super nitpick: `cl` is short for "classified listing" I guess. I spent a lot of time in the past refactoring the code so we only have one domain name left, which is "listing", so it'd be great if we stop using `cl` and similar names to refer to them.